tests/env-vars.spec.ts

```diff
@@ -0,0 +1,198 @@
+import { BuildStatus, SauceRegion, getApi } from '@saucelabs/visual';
+import {
+  RE_VISUAL_BUILD_ID,
+  RE_VISUAL_BUILD_LINK,
+  SAUCE_VISUAL_BRANCH,
+  SAUCE_VISUAL_BUILD_NAME,
+  SAUCE_VISUAL_DEFAULT_BRANCH,
+  SAUCE_VISUAL_PROJECT,
+  waitStatusForBuild,
+} from './utils/helpers';
+import { execute } from './utils/process';
+import { FileHandle } from 'fs/promises';
+import { randomBytes } from 'crypto';
+
+const region = 'us-west-1' as SauceRegion;
+
+const visualApi = getApi({
+  region,
+  user: process.env.SAUCE_USERNAME!,
+  key: process.env.SAUCE_ACCESS_KEY!,
+});
+
+const customId = randomBytes(20).toString('hex');
+
+let fileOutput: FileHandle | undefined;
+let dockerOutput = '';
+let buildId = '';
+let externalBuildId = '';
+
+describe('Env var tests', () => {
+  it(
```

Comment:
This should probably be in a `beforeAll` more than an `it`.

We may also want an `afterAll` to close the build.